azure-pipelines/java-16-install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+
+wget -c --header "Cookie: oraclelicense=accept-securebackup-cookie" https://download.oracle.com/otn-pub/java/jdk/16.0.2%2B7/d4a915d82b4c4fbb9bde534da945d746/jdk-16.0.2_linux-x64_bin.tar.gz
```

Comment:
it'd be nice if there were some comments here or out on the step letting people know what considerations went into this implementation, links to docs you used, how they can go about updating it to the next version of java 16, etc. that should be your frame of mind anytime you implement something non-obvious.
we're running out of time so i won't block it based on this.